sunpy/net/hek/hek.py

```diff
@@ -73,7 +74,12 @@ def _download(self, data):
             results.extend(result['result'])
 
             if not result['overmax']:
-                return HEKTable(results)
+                if len(results) > 0:
+                    return HEKTable(results)
+                else:
+                    log.info('Search returned no results')
```

Comment:
I am not sure if there is value in logging something here, I mean it's obvious there is no results because the table is empty??? :grin:

Reply:
That's true I was trying to be explicit but it probably overkill I'll remove it 👍